lia-app-uikit/View Controllers/DetailViewController.swift

```diff
@@ -16,10 +37,11 @@ class DetailViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         setupUI()
+        loadImage()
```

Comment:
Call in init already if possible? If possible image will have some more time to download before view is shown